policy/modules/services/munin.te

```diff
@@ -260,7 +260,7 @@ optional_policy(`
 ')
 
 optional_policy(`
-	fstools_exec(disk_munin_plugin_t)
+	fstools_run(disk_munin_plugin_t, system_r)
```

Comment:
I think `fstools_domtrans()` is a better fit here. The `run` interfaces for intended for interactive running by users.